figures/generate-figures.sh

```diff
@@ -148,16 +150,15 @@ cp ${analyses_dir}/mutational-signatures/plots/cns/hypermutator_sigs_heatmap_leg
 
 # Create directory
 mkdir -p pdfs/fig5/panels
-
+# # This directory is also created here because one of the scripts also generates S6 panels
+mkdir -p pdfs/supp/figs6/panels
```

Comment:
just a general question about ordering - maybe these directories can be all made at the top of the script so that it is easier to follow which are being made altogether?

Reply:
Sure!